app/services/data_sync/elasticsearch/user.rb

```diff
@@ -0,0 +1,51 @@
+module DataSync
+  module Elasticsearch
+    class User
+      RELATED_DOCS = %i[
+        articles
+        reactions
+        podcast_episodes
+        chat_channel_memberships
+        comments
+      ].freeze
+
+      SHARED_USER_FIELDS = %i[
+        username
+        name
+        pro
+        profile_image_url
+      ].freeze
+
+      attr_accessor :user, :updated_fields
+
+      delegate :articles, :chat_channel_memberships, :comments, to: :@user
+
+      def initialize(user, updated_fields)
+        @user = user
+        @updated_fields = updated_fields.deep_symbolize_keys
+      end
+
+      def sync_documents
```

Comment:
do we plan to have more methods? otherwise we can follow the `.call` convention we have in other services. Totally not a blocker :D

Reply:
Huh, yeah I guess we are not going to be calling anything else. One thing I like about a method like this is that its more clear what it is doing. We are syncing documents, not settings, not mappings, only documents.